README.md

```diff
@@ -1,2 +1,105 @@
 # SeineSailor
-Help developers navigating through the codebase smoothly
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![GitHub](https://img.shields.io/github/last-commit/SeineAI/SeineSailor/main?style=flat-square)](https://github.com/SeineAI/SeineSailor/commits/main)
+
+## Overview
+
+SeineSailor is an AI-powered copilot for project managers, designed to assist with code reviews, project discussions,
+and knowledge transfer. It leverages the power of WatsonX's `llama3` and `mixt-8x7B` models to provide intelligent and
+context-aware support throughout the development lifecycle.
+
+## Features
+
+- **PR Review and Summarization**: SeineSailor can be summoned to review pull requests and provide a summary of the
+  changes when mentioned in a PR comment.
+- **Project Discussion Participation**: SeineSailor can actively participate in project discussions and brainstorming
+  sessions when invited using the `@SeineSailor` mention in issues or other discussion forums.
+- **Knowledge Transfer**: SeineSailor maintains a knowledge base of the project's development history and can assist new
+  developers by answering questions about the codebase, architecture, and function implementations.
+- **Customizable Prompts**: Tailor the `system_message`, `summarize`, and other prompts to focus on specific aspects of
+  the review process or change the review objective.
+
+To use SeineSailor, you need to add the provided YAML file to your repository and configure the required environment
+variables, such as `GITHUB_TOKEN` and `WATSONX_API_KEY`. For more information on usage, examples, and contributing,
+refer to the sections below.
+
+## Installation
+
+Add the following file to your repository at `.github/workflows/seinesailor.yml`:
+
+```yaml
+name: SeineSailor
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+  discussions: write
+
+on:
+  issue_comment:
+    types: [ created ]
+  pull_request:
+    types: [opened, synchronize, reopened]
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+  pull_request_review_comment:
+    types: [ created ]
+  discussion:
+    types: [ created ]
+
+concurrency:
+  group:
+    ${{ github.repository }}-${{ github.event.number || github.head_ref ||
+    github.sha }}-${{ github.workflow }}-${{ github.event_name ==
+    'pull_request_review_comment' && 'pr_comment' || 
+    github.event_name == 'issue_comment' && 'issue' || 
+    github.event_name == 'discussion' && 'discussion' }}
+  cancel-in-progress: true
+  
+jobs:
+  seinesailor:
+    if: contains(github.event.comment.body, '@SeineSailor')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: SeineAI/SeineSailor@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          IBM_CLOUD_API_KEY: ${{ secrets.IBM_CLOUD_API_KEY }}
+          WATSONX_PROJECT_ID: ${{ secrets.WATSONX_PROJECT_ID }}
+```
+
+### Environment Variables
```

Comment:
@SeineSailor test run

Reply:
🤖   SeineSailor



@wenboown: This is a test run of SeineSailor. I have reviewed the pull request and provided a summary of the changes. The updated README.md file includes an overview of SeineSailor, its features, installation instructions, usage examples, and contributing guidelines.

<!-- This is an auto-generated reply by OSS SeineSailor -->